scripts/linters/js_ts_linter.py

```diff
@@ -220,6 +228,57 @@ def _compile_ts_file(self, filepath, dir_path):
             dir_path, os.path.basename(filepath).replace('.ts', '.js'))
         return compiled_js_filepath
 
+    def _check_any_type(self):
+        """Checks if the type of any variable is declared as any
+        in typescript files.
+        """
+
+        if self.verbose_mode_enabled:
+            python_utils.PRINT('Starting any type check')
+            python_utils.PRINT('----------------------------------------')
+
+        files_to_check = self.all_filepaths
+        patterns_to_match = [r':\ *any', r'^\ *any']
+        stdout = sys.stdout
+
+        with linter_utils.redirect_stdout(stdout):
+            failed = False
```

Comment:
Do you need `failed`? Can you just check if `summary_messages` is not empty?

Reply:
@nishantwrp What about this? Please reply to all comments before you request another review.

Reply:
Oh, sorry @vojtechjelinek. Missed this. Actually are you suggesting that deciding that the check has passed if summary errors is empty?

If that's the case, I can't think we can do that because there is a success message in summary errors if it is has passed.

Reply:
But success message is added as the last item, so it should be okay to check on line 269 if the summary_messages are empty and then add the message.

Reply:
I mean, success_messages is always gonna be empty on line 269

Reply:
Oh, I see sorry.